FreeRTOS-Plus/Source/Application-Protocols/platform/freertos/transport/include/tls_freertos.h

```diff
@@ -94,31 +96,26 @@ struct NetworkContext
 typedef struct NetworkCredentials
 {
     /**
-     * @brief Set this to a non-NULL value to use ALPN.
-     *
-     * This string must be NULL-terminated.
+     * @brief Set this to a NULL-terminated list of supported protocols,
+     * in decreasing preference order to use ALPN.
```

Comment:
Let's make the same change here
```suggestion
     * @brief To use ALPN, set this to a NULL-terminated list of supported
     * protocols in decreasing order of preference.
```